guardian/cdk-cli
# docs: add migration  instructions and gotchas

## What does this change?


**To be merged after #210**

This PR adds some extra instructions and gotchas to the README for migrating an application.

## How to test


Run the `How to migrate from Cloudformation YAML/JSON` example in the README and see if it successfully generates some Cloudformation!

## How can we measure success?


We can successfully initialise a CDK directory and get the parameters of an existing CFN YAML file captured in the library.
